ui/package.json

```diff
@@ -12,8 +12,8 @@
     "start": "craco start",
     "build": "craco build",
     "test:unit": "craco test",
-    "test": "yarn lint && yarn test:unit && yarn test:size",
     "test:size": "yarn build && bundlesize",
+    "test": "yarn lint && yarn test:unit --no-watch && yarn test:size",
```

Comment:
why do we need ``--no-watch``?

Reply:
There is a limit on the number of files you can watch simultaneously and inside the docker container it seems to be lower. So the tests fail.

Reply:
@harunurhan will this affect local testing?

Reply:
no it won't